files/mono-build-win32.sh

```diff
@@ -17,13 +17,13 @@ export PATH="$(pwd)/.bin/:$PATH"
 ./autogen.sh $@ --disable-boehm --with-mcs-docs=no HOST_PROFILE=win32
 echo '#define HAVE_STRUCT_SOCKADDR_IN6 1' >> config.h
 pushd mcs/jay
-make CC=gcc
+make CC=gcc BUILD_PLATFORM=win32
 popd
 
 for dir in external/roslyn-binaries/Microsoft.Net.Compilers/[0-9]*; do
   MONO_PATH="$(winepath -w $(pwd)/${dir});${MONO_PATH}"
 done
 export MONO_PATH
 
-make -j
-make install
+make -j SHELL='bash -x'
+make install SHELL='bash -x'
```

Comment:
Isn't `make` verbose enough as is?

Reply:
I'll look again, but when I was debugging this, it was just stopping with an error, but no command. It might be fine without.